src/lib.rs

```diff
@@ -2554,7 +2541,7 @@ impl Build {
             traditional
         };
 
-        let cl_exe = windows_registry::find_tool(target, "cl.exe");
+        let cl_exe = windows::find_tools::find_tool(target, "cl.exe");
```

Comment:
there's no need to change this line:

```suggestion
        let cl_exe = windows_registry::find_tool(target, "cl.exe");
```

Reply:
done